### PR TITLE
Add name and website to OrganizationEntity

### PR DESCRIPTION
**Pull request by lex111**

### CODE REVIEW

**Comment by lex111** [comment]
ping @ovr

**Comment by ovr** [comment]
pong @lex111

File: github-swagger.json [hunk 1]
```diff
@@ -1050,7 +1050,9 @@
         "id",
         "login",
         "avatar_url",
-        "description"
+        "description",
+        "name",
+        "website"
```
Comment (ovr) [inline github-swagger.json]: https://api.github.com/orgs/socialconnect

image

Looks like, website doesnot exist, there is `blog` field

Comment (lex111) [reply github-swagger.json]: Sorry about that: I mixed up with GraphQL :(
Fixed.

**Comment by ovr** [comment]
Thank you!

**Pull request merged**